.gitignore

```diff
@@ -7,56 +7,28 @@ __pycache__/
 .vscode/
 .vs/
 .idea/
-CMakeLists.txt
-cmake-build-debug
-venv/
-.venv/
-tags
+.cache/
+compile_commands.json
 
 # Project-specific ignores
-*.z64
-*.n64
-*.v64
-*.sra
-*.bin
-*.elf
-*.flash
-archive/
+.make_options.mk
+extracted/
 build/
-baserom/
-decomp/
+expected/
 asm/
 data/
-extracted/
-expected/
-nonmatchings/
+docs/doxygen/
+*.z64
+*.n64
+*.v64
 
-# Tool artifacts
-tools/ido5.3_compiler/
-tools/ido7.1_compiler/
-tools/qemu-mips
-tools/ido_recomp/* binary
+# Tools
+.venv/
 ctx.c
+tools/*dSYM/
 graphs/
-*.c.m2c
```

Comment:
`.c.m2c` is a cache file that m2c generates, so I think it is fine to keep it, even if there's little to none decomp work to do. Specially because we are keeping `ctx.c` on the gitignore